app/controllers/website_members_controller.rb

```diff
@@ -1,14 +1,12 @@
 class WebsiteMembersController < ApplicationController
 	load_and_authorize_resource
   def index
-    @members = current_website.website_members
+    @website_members = current_website.website_members.where("user_id IS NOT NULL")
```

Comment:
We can remove this. Now we have invites table and there is no case when website_member is created without an user_id.